config.ld

```diff
@@ -13,28 +13,29 @@ title = 'Lunatik API Documentation'
 -- LDoc will recursively scan directories.
 -- By manually specifying order, we ensure menu order.
 file = {
-    './lib/luacompletion.c',
-    './lib/luadata.c',
-    './lib/luadevice.c',
-    './lib/luafib.c',
-    './lib/luafifo.c',
-    './lunatik_core.c',
-    './lib/lunatik/runner.lua',
-    './lib/lualinux.c',
-    './lib/mailbox.lua',
-    './lib/net.lua',
-    './lib/luanetfilter.h',
-    './lib/luanetfilter.c',
-    './lib/luanotifier.c',
-    './lib/luaprobe.c',
-    './lib/luarcu.c',
-    './lib/luasocket.c',
-    './lib/socket/inet.lua',
-    './lib/luasyscall.c',
-    './lib/syscall/table.lua',
-    './lib/luathread.c',
-    './lib/luaxdp.c',
-    './lib/luaxtable.c',
+	'./lib/luacompletion.c',
+	'./lib/luacrypto_shash.c',
+	'./lib/luadata.c',
+	'./lib/luadevice.c',
+	'./lib/luafib.c',
+	'./lib/luafifo.c',
+	'./lunatik_core.c',
+	'./lib/lunatik/runner.lua',
+	'./lib/lualinux.c',
+	'./lib/mailbox.lua',
+	'./lib/net.lua',
+	'./lib/luanetfilter.h',
+	'./lib/luanetfilter.c',
+	'./lib/luanotifier.c',
+	'./lib/luaprobe.c',
+	'./lib/luarcu.c',
+	'./lib/luasocket.c',
+	'./lib/socket/inet.lua',
+	'./lib/luasyscall.c',
+	'./lib/syscall/table.lua',
+	'./lib/luathread.c',
+	'./lib/luaxdp.c',
+	'./lib/luaxtable.c',
```

Comment:
Please, let's make formatting changes in separate PRs.. it's make it harder to review